src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializerVersionAttribute.cs

```diff
@@ -2,7 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if XMLSERIALIZERGENERATOR
+namespace Microsoft.XmlSerializer.Generator
+#else
 namespace System.Xml.Serialization
+#endif
 {
```

Comment:
Does Sgen need to include this file? As I recalled, you changed the code to use System.Xml.Serialization.XmlSerializerVersionAttribute explicitly. I think we may not need this.

The same for `XmlSerializerAssemblyAttribute`.